### ISSUE TITLE
Speed on GeForce GTX 1080

### ISSUE THREAD

**Issue by srp-31** [victordibia/handtracking#26]
Hi

I am running on GeForce GTX 1080(notebook GPU). I get a speed of 15 fps stable with image display and about 30 fps without display. I have a feeling that with my graphics card the performance should be better. Does anyone have any comments about the same. I have another object detector trained for yolo and there too the speeds I achieve are lesser than widely reported.

Any help would be greatly appreciated.

thanks
shreeraman

**Comment by srp-31** [comment]
Hi

I found that the isssue was due tobalanced power mode on windows. With high performance mode enabled the speed was better.

20 fps with displlay and 30 fps again without display.

When the image grabbing  from the camera is separated onto another thread the power mode enabled gives fps in the rnage of 50(I verified this by tapping the time before detect_objects and draw_box_on_image are called and after they are called). The two functions in total take 0.015-0.02 seconds and hence the 50 fps frame rate.

I am using a MSI GT73VR GRF Tiatn Proc gaming PC. The dragon centre app from msi is used to change the power option.  

I am closing the issue since the fps is as per expectation. The original SSD paper reported 59fps with Titan X with 11 TFLOPS. My GeFroce GTX 1080 has 6 flops. I made rough calculation based only on that-6*59/11=42 fps. i receive close to 50fp which could be due to over clocking with high perfromance mode. ( I am guessing this a is valid approximate calculaation to make)

Thanks and hope this helps someone out there :)
shreeraman
